src/app/game/question-display/page.tsx

```diff
@@ -1,30 +1,51 @@
 import styles from "./question-display.module.scss";
 import { Game } from "../../../lib/model";
 import { useState } from "react";
+import { useRouter } from "next/navigation";
 
 export default function QuestionDisplay({
+  timer,
   game,
   callback,
 }: {
+  timer: number;
   game: Game;
   callback: (correct: boolean) => void;
 }) {
   let currentQuestion = game.questions[game.gameLevel - 1];
   const [selectedAnswer, setSelectedAnswer] = useState<number | null>(null);
   const [correctAnswer, setCorrectAnswer] = useState<number | null>(null);
 
+  const router = useRouter();
```

Comment:
I think the best way to handle this is outside this component.

Reply:
Ok Sir